src/client/public/scripts/dash/utils/newly-actionable-issues.ts

```diff
@@ -0,0 +1,24 @@
+import {Issue} from '../../../../../types/api.js';
+
+export function newlyActionableIssues(
+    newList: Issue[], oldList: Issue[]): string[] {
+  const result: string[] = [];
+
+  const oldActionableIssues: {[issueId: string]: Issue} = {};
```

Comment:
I think you only need the issueIds. `oldIds = oldList.map((issue: Issue) => issue.id)`